src/goto-analyzer/static_verifier.cpp

```diff
@@ -27,6 +27,31 @@ struct static_verifier_resultt
   irep_idt function_id;
 };
 
+static static_verifier_resultt::statust
```

Comment:
Wouldn't a declaration also do the trick? (Just to reduce churn.)

Reply:
Alas no.  The problem is...

1. I want to move the JSON and XML generation code for a single result out of the methods.  Having them as member functions of `static_verifier_resultt` seems consistent with the code base.
2. They use the `message` function, so it must appear be declared after the `statust` enum but before `output_(xml|json)`.
3. While the `enum` is declared in `static_verifier_result` this is not possible without making it a static member function which doesn't really reduce churn and misses the point of the separation really.

I see your point about churn but i have tried to minimise it and I don't think it is too bad.